src/mpyl/reporting/targets/jira.py

```diff
@@ -227,7 +227,7 @@ def send_report(
             self.__move_ticket_forward(ticket)
 
             user_email = results.run_properties.details.user_email
-            if user_email:
+            if user_email and not ticket.user_email:
```

Comment:
`ticket.user_email` signifies the reporter or creator of the ticket. That's always set.
The current logic should already only assign unassigned tickets:
https://github.com/Vandebron/mpyl/blob/f325b7b152ab86f536bd7656f8a13f8176838037/src/mpyl/reporting/targets/jira.py#L238-L239